### ISSUE TITLE
Xenko still shows privacy agreement popup even when running 3.0.0.5-dev

### ISSUE THREAD

**Issue by aurodev** [xenko3d/xenko#204]
**Release Type**: Official Release/GitHub (please choose appropriate option)

**Version**: Version number and/or git branch
3.0.0.5-dev Master
**Platform(s)**: Does the problem occur on Windows, Android...?
Windows 8.1
**Describe the bug**
Xenko still shows privacy agreement popup even when running a github build for the first time. This occurs when the Xenko Installer version has been removed.

**To Reproduce**
Build the 3.0.0.5 master branch from github
Run Xenko.GameStudio.exe

**Expected behavior**
An MIT project should not require acceptance of a privacy agreement.

**Screenshots**

**Log and callstacks**

**Additional context**

**Comment by xen2** [comment]
In the current code, Xenko still send some anonymous usage metrics even when run from GitHub.

Do you think that's something that should be disabled except in official builds? (at the risk of loosing lot of useful statistics for us)

**Comment by aurodev** [comment]
In the current climate where the public is somewhat suspicious of data collection... yes.
Not that big a deal actually but perhaps it could be made opt-in to help Xenko instead of "must accept"
IMHO

**Comment by Kryptos-FR** [comment]
@aurodev good point. We should make it opt-in and maybe explain exactly what we are collecting.
